receiver/prometheusreceiver/internal/metricfamily.go

```diff
@@ -319,3 +334,55 @@ func (mf *metricFamily) appendMetric(metrics pmetric.MetricSlice) {
 
 	metric.MoveTo(metrics.AppendEmpty())
 }
+
+func (mf *metricFamily) addExemplar(l labels.Labels, e exemplar.Exemplar) {
+	gk := mf.getGroupKey(l)
+	mg := mf.groups[gk]
+	if mg == nil {
+		return
+	}
+	es := mg.exemplars
+	convertToOtlpExemplar(e, es.AppendEmpty())
+}
+
+func convertToOtlpExemplar(pe exemplar.Exemplar, e pmetric.Exemplar) {
+	e.SetTimestamp(timestampFromMs(pe.Ts))
+	e.SetDoubleVal(pe.Value)
+	e.FilteredAttributes().EnsureCapacity(len(pe.Labels))
+	for _, lb := range pe.Labels {
+		switch strings.ToLower(lb.Name) {
+		case "trace_id", "traceid":
+			var tid [16]byte
+			err := decodeAndCopyToLowerBytes(tid[:], []byte(lb.Value))
+			if err == nil {
```

Comment:
Why ignoring the error?

Reply:
Setting traceid, spanid as attributes in case of err.